### PR TITLE
Access spoke api using cluster-proxy

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

**Comment by qiujian16** [comment]
/hold

wait for clusteradm PR to be merged

/assign @xuezhaojun

File: solutions/access-remote-api/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Create managed-service account on hub
+
+```
+kubectl apply -f manifests/managed-sa.yaml
+```
+
+create a clusterrolebinding on managed cluster to set permission for this service account
+
+```
+clusteradm create work rbac -f manifests/clusterrolebinding.yaml
```
Comment (xuezhaojun) [inline solutions/access-remote-api/README.md:38]: Should we add cluster name flag?
```
clusteradm create work rbac -f manifests/clusterrolebinding.yaml --cluster=cluster1
```

File: solutions/access-remote-api/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+helm install \
+    -n open-cluster-management-addon --create-namespace \
+    cluster-proxy ocm/cluster-proxy 
+```
+
+Install managed-serviceaccount addon:
+
+```
+helm install \
```
Comment (yue9944882) [inline solutions/access-remote-api/README.md:22]: probably we should let the user confirm the addon installation status by:

```
clusteradm get addon cluster-proxy
```

**Pull request merged**